server/php/UploadHandler.php

```diff
@@ -1264,20 +1272,23 @@ protected function download() {
                 $redirect_header = 'X-Accel-Redirect';
                 break;
             default:
-                return $this->header('HTTP/1.1 403 Forbidden');
+                $this->header('HTTP/1.1 403 Forbidden');
+                return;
```

Comment:
Any reason why you broke this up into multiple lines?
The `header` function already returns `null`.